examples/stargate-jwt-auth/docker-compose.yml

```diff
@@ -0,0 +1,79 @@
+version: '2'
+
+services:
+  cass-1:
+    image: cassandra:${CASSTAG}
+    networks:
+      - backend
+    ports:
+      - 9044:9042
+    mem_limit: 2G
+    environment:
+      - HEAP_NEWSIZE=128M
+      - MAX_HEAP_SIZE=1024M
+      - CASSANDRA_SEEDS=cass-1
+      - CASSANDRA_CLUSTER_NAME=sg-jwt-${CASSTAG}
+  cass-2:
+    image: cassandra:${CASSTAG}
+    networks:
+      - backend
+    mem_limit: 2G
+    depends_on:
+      - cass-1
+    environment:
+      - HEAP_NEWSIZE=128M
+      - MAX_HEAP_SIZE=1024M
+      - CASSANDRA_SEEDS=cass-1
+      - CASSANDRA_CLUSTER_NAME=sg-jwt-${CASSTAG}
+  cass-3:
+    image: cassandra:${CASSTAG}
+    networks:
+      - backend
+    mem_limit: 2G
+    depends_on:
+      - cass-1
+    environment:
+      - HEAP_NEWSIZE=128M
+      - MAX_HEAP_SIZE=1024M
+      - CASSANDRA_SEEDS=cass-1
+      - CASSANDRA_CLUSTER_NAME=sg-jwt-${CASSTAG}
+  stargate-jwt:
+    image: stargate:${SGTAG}
+    container_name: stargate-jwt
+    depends_on: 
+      - cass-1
+    networks:
+      - backend
+    ports:
+      - 8080:8080
+      - 8081:8081
+      - 8082:8082
+      - 8084:8084
+      - 9045:9042
+    mem_limit: 2G
+    environment:
+      - JAVA_OPTS=-XX:+CrashOnOutOfMemoryError -Xmx750M -Xms64M -Dstargate.auth_id=AuthJwtService -Dstargate.auth.jwt_provider_url=http://keycloak:4444/auth/realms/stargate/protocol/openid-connect/certs
```

Comment:
If you want to use the JWT with cql you'll also need `-Dstargate.cql_use_auth_service=true`

Reply:
fixed